### ISSUE TITLE
Casing is wrong in example

### ISSUE THREAD

**Issue by azimuthdeveloper** [kensnyder/quill-image-resize-module#40]
When using Quill on a html page without webpack or anything, the example says to do the following to load the module.

   ImageResize: {
            // See optional "config" below
        }

This doesn't work and throws an error saying it can't find a module by that name.  Changing ImageResize to imageResize makes it work though.

**Comment by sireeshavnv** [comment]
Yes I found that same issue. Fiddled around for sometime before being able to make it work.
